recipes/_alfrescoproperties-attributes.rb

```diff
@@ -35,22 +35,19 @@
 node.default['artifacts']['alfresco-s3-connector']['destination'] = node['alfresco']['amps_folder']
 
 # ssl-db-creds
-node.default['artifacts']['ssl-db-creds']['url'] = "http://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem"
-node.default['artifacts']['ssl-db-creds']['destination'] = "/tmp/rds-combined-ca-bundle.pem"
-node.default['artifacts']['ssl-db-creds']['owner'] = "tomcat"
+node.default['artifacts']['ssl-db-creds']['url'] = 'http://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem'
+node.default['artifacts']['ssl-db-creds']['destination'] = '/tmp/rds-combined-ca-bundle.pem'
+node.default['artifacts']['ssl-db-creds']['owner'] = 'tomcat'
 node.default['artifacts']['ssl-db-creds']['enabled'] = false
 
-# HTTP default pool size
-node.default['alfresco']['properties']['httpclient.max-connections'] = "20"
```

Comment:
This hasn't been deleted but moved with the S3 attributes, as @steverigby mentioned that this is only relevant for the S3 connector.